src/components/PipelineRunDetailsView/MetadataList.tsx

```diff
@@ -14,8 +16,8 @@ const MetadataList: React.FC<MetadataListProps> = ({ metadata }) => {
       <LabelGroup>
         {labelArray.map((label, index) => (
           <FlexItem key={index}>
-            <Label color="blue" isTruncated={label.toString().length > 100}>
-              {label.toString().replace(',', '=')}{' '}
+            <Label color="blue" className="metadata-list__label">
```

Comment:
Is this still necessary `className="metadata-list__label"`?
Seems like the PF component should take care of it.

Reply:
max-width doesn't work without this

Reply:
Max width doesn't seem necessary and in fact can have negative results by prematurely limiting the width when there is enough space in the column available and 35vw is smaller.

Reply:
updated
<img width="1198" alt="Screenshot 2023-03-03 at 4 33 56 PM" src="https://user-images.githubusercontent.com/24852534/222717454-3e943a39-99a7-4948-acf9-9ded4740a955.png">
<img width="585" alt="Screenshot 2023-03-03 at 4 34 06 PM" src="https://user-images.githubusercontent.com/24852534/222717479-c4391827-cb7a-49e5-8e4f-60134ff707c8.png">